apis/config/v1beta1/defaults.go

```diff
@@ -41,6 +39,7 @@ const (
 	defaultPodsReadyTimeout                             = 5 * time.Minute
 	DefaultQueueVisibilityUpdateIntervalSeconds int32   = 5
 	DefaultClusterQueuesMaxCount                int32   = 10
+	DefaultJobFrameworkName                             = "batch/job"
```

Comment:
```suggestion
	defaultJobFrameworkName                             = "batch/job"
```

Is this constant used from external packages?

Reply:
Good catch, un-exporting